src/locales/ru/stream.ftl

```diff
@@ -117,6 +117,7 @@ comments-permalinkPopover-permalinkToComment =
 comments-permalinkButton-share = Поделиться
 comments-permalinkButton =
   .aria-label = Поделиться
+comments-permalink-copyLink = Скопировать ссылку
```

Comment:
@Emptyfruit, did you want to provide a translation for `comments-permalink-linkCopied` as well?

Reply:
probably. Thank you, you had those keys missing :)